### ISSUE TITLE
furnace : can take picture of toxic cellar player saying otherwise.

### ISSUE THREAD

**Issue by N-G-Neer** [loiste-interactive/infra-issues#802]
https://youtu.be/LcQTC3Qohrg?t=751

Mark says that SBS is abandoned yet the toxic cellar till counts as a photospot. Also maybe a line of dialogue should be added ?
"Stalburg Steel. This place is abandoned. I don't need to take any pictures here. Well, except for interesting documents, of course..."

**Comment by aleksijuvani** [comment]
I feel like the whole logic of what counts as a picture in here is really shaky and doesn't make much sense. Maybe Mark could say something like that he doesn't need to take any pictures of _structural damage_ in here.

**Comment by N-G-Neer** [comment]
Yeah and there are some more inconsistencies as well concerning the mushrooms. For example, in the first level of Part 2, when you arrive in the brick activated button puzzle, you can't take pictures of the mushroom clusters, meanwhile, you can take pictures of every single little group in the scientists living quarters in bunker.

**Comment by sebastiannielsen** [comment]
Yeah, in the sewer level you can usually not photo every mushroom clusters, only those "large enough" to be a problem.

But it also makes sense that mushroom clusters in the living quarters, as these mushrooms are growing in a human area and thats not good, regardless of the size of the mushrooms.

**Comment by N-G-Neer** [comment]
Don't know if this is the same photospot trigerring from a different angle, but here goes : 
![20170926040040_1](https://user-images.githubusercontent.com/15633807/30861101-f373a8c2-a2c9-11e7-857f-25f724eb24a7.jpg)
